build.gradle.kts

```diff
@@ -108,6 +108,15 @@ kotlin {
             pom { name = "${project.name}-wasm-js" }
         }
     }
+    @OptIn(ExperimentalWasmDsl::class)
+    wasmWasi {
+        nodejs()
+
+        mavenPublication {
```

Comment:
not sure if this is required at all - native targets misses such block, but still I copied it from `wasm-js`